app/components/application_component.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class ApplicationComponent < ViewComponent::Base
-  delegate :can?, to: :controller
+  delegate :can?, :current_user, to: :controller
```

Comment:
You can remove this addition.

Reply:
@jcoyne If I remove the delegation, there are failures due to https://github.com/sul-dlss/argo/pull/4592/files#diff-6c1bb310d0eac46c88fec511636f41bd4577cb38e6dbf2455a6f4db8112c4632R13-R15

If I revert *that* change so that we go back to getting the current user from `helpers`, that leads to this:

![Screenshot from 2024-08-05 13-36-36](https://github.com/user-attachments/assets/13e42262-9321-4c94-8db2-3877bde05915)